tests/integration/llm/prepare.py

```diff
@@ -962,6 +962,29 @@
         "option.rolling_batch": "auto",
         "option.max_rolling_batch_size": 213,
         "option.model_loading_timeout": 1800
+    },
+    "trtllm-meta-llama3-8b-fp8": {
+        "engine": "Python",
+        "option.task": "text-generation",
+        "option.model_id": "s3://djl-llm/llama-3-8b-hf/",
+        "option.rolling_batch": "trtllm",
+        "option.tensor_parallel_degree": 4,
+        "option.quantize": "fp8"
+    },
+    "trtllm-mistral-7b-instruct-v0.3": {
+        "engine": "Python",
+        "option.task": "text-generation",
+        "option.model_id": "mistralai/Mistral-7B-Instruct-v0.3",
```

Comment:
We'll want to change these to the s3 locations. Let's use `s3://djl-llm/mistral-7b-instruct-v03/` here and for the other mistral config below.